method and results - single label function section (#22)
greenelab/text_mined_hetnet_manuscript: content/04.results.md

The last most column consists of pooling every relationship specific label function and proceeding as above.
](https://raw.githubusercontent.com/greenelab/snorkeling/master/figures/label_sampling_experiment/transfer_test_set_auprc.png){#fig:gen_model_auprc}

### Random Label Function Gen Model Analysis
![
A grid of area under the receiver operating curve (AUROC) for each edge type.
Each plot consists of adding a single label function on top of the baseline model.

> This label function emits a positive (top row) or negative (bottom row) label at specified frequencies.

Reviewer: ```suggestion
This label function emits a positive (top row) or negative (bottom row) label at specified frequencies, and performance at zero is equivalent to not having a randomly emitting label function.
```

> The error bars represents confidence intervals for emission frequency.

Reviewer: How big of confidence intervals? 90% 95%? Something else?

Author: 95% will add to both sentences.

Reviewer: ok 👍 

Confidence intervals also look like they relate to what's on the Y axis (instead of emission freq which is X-axis). After changing these things. Feel free to merge.

](https://raw.githubusercontent.com/danich1/snorkeling/f8962788e462b783be05a6dec5eec7fe0f0259e7/figures/gen_model_error_analysis/transfer_test_set_auroc.png){#fig:random_label_function_auroc}

> Due to the one label function anomaly (mentioned in Results:Generative Model Using Randomly Sampled Label Functions), we evaluated the effects of adding random noise on top of the generative model, trained using only distant supervision label functions.

Reviewer: ```suggestion
We observed that including one label function of a mismatched type to distant supervision often improved performance, so we evaluated the effects of adding a random label function in the same setting.
```

> In general, adding random noise does not improve performance (Figures {@fig:random_label_function_auprc} and {@fig:random_label_function_auprc}).

Reviewer: ```suggestion
We found that adding random noise did not usually improve performance (Figures {@fig:random_label_function_auprc} and {@fig:random_label_function_auprc}).
```

> One exception to this trend is CbG, which had improved AUPR performance (Figure {@fig:random_label_function_auprc}).

Reviewer: ```suggestion
For the CbG edge type we did observe slightly increased performance via AUPR (Figure {@fig:random_label_function_auprc}).
However, in general the performance changes were smaller than those observed with mismatched label types.
```

![
A grid of area under the precision recall curve (AUPR) for each edge type.
Each plot consists of adding a single label function on top of the baseline model.
This label function emits a positive (top row) or negative (bottom row) label at specified frequencies.

> The error bars represents confidence intervals for emission frequency.

Reviewer: Same question here about the type of CIs being plotted.
